### PR TITLE
Added copyright link to tos page

### PR DESCRIPTION
**Pull request by a-gene-i**
@caesar2164 @stvstnfrd

### CODE REVIEW

File: lagunita/lms/templates/static_templates/tos.html [hunk 1]
```diff
@@ -60,7 +62,7 @@ <h1 id="tos" style="text-align: left;">${_('Welcome to the Stanford Online Learn
             ).format(
                 pii_link_start=pii_link,
                 research_link_start='<a href="https://lytics.stanford.edu/research/"><strong>',
-                link_end='</strong></a>'
+                link_end=link_end
```
Comment (stvstnfrd) [inline lagunita/lms/templates/static_templates/tos.html]: https://github.com/Stanford-Online/edx-platform/blob/master/openedx/stanford/style/python/collections-trailing-comma.markdown

File: lagunita/lms/templates/static_templates/tos.html [hunk 2]
```diff
@@ -522,6 +524,13 @@ <h2 id="tos">${_('Terms of Service for Stanford Online Courses')}</h2>
                 date=effective_date
             ) | n }
         </strong></p>
+
+        <p>
+        ${_('''Please visit {copyright_link}Copyright Complaints{link_end} for information on how to submit a complaint.''').format(
```
Comment (stvstnfrd) [inline lagunita/lms/templates/static_templates/tos.html]: https://github.com/Stanford-Online/edx-platform/blob/master/openedx/stanford/style/python/docstrings.markdown

File: lagunita/lms/templates/static_templates/tos.html [hunk 3]
```diff
@@ -522,6 +524,13 @@ <h2 id="tos">${_('Terms of Service for Stanford Online Courses')}</h2>
...
+            copyright_link=copyright_link,
+            link_end=link_end
```
Comment (stvstnfrd) [inline lagunita/lms/templates/static_templates/tos.html]: https://github.com/Stanford-Online/edx-platform/blob/master/openedx/stanford/style/python/collections-trailing-comma.markdown

**Review by stvstnfrd — CHANGES_REQUESTED** [review]
A couple of style notes.
Also, can you include a screenshot of what this looked like _before_ and _after_ the change?

@caesar2164 Do we need this change on the other themes too?

**Comment by caesar2164** [comment]
@stvstnfrd, we do NOT need to add this to the other themes cause @kluo nicely symlinked them! 13b20733759d2d28c2a598c12ce237b28e1dad8f

File: lagunita/lms/templates/static_templates/tos.html [hunk 4]
```diff
@@ -522,6 +524,13 @@ <h2 id="tos">${_('Terms of Service for Stanford Online Courses')}</h2>
                 date=effective_date
             ) | n }
         </strong></p>
+
+        <p>
+        ${_("""Please visit {copyright_link}Copyright Complaints{link_end} for information on how to submit a complaint.""").format(
```
Comment (stvstnfrd) [inline lagunita/lms/templates/static_templates/tos.html]: @glewis17 Is this the text requested on the ticket?

Comment (a-gene-i) [reply lagunita/lms/templates/static_templates/tos.html]: @stvstnfrd The text as requested on the ticket doesn't make sense for Copyright Complaints.
@mondiaz Are you ok with this text?

**Comment by mondiaz** [comment]
@glewis17 Thanks for catching that the requested text isn't directly applicable. Let me get back to you on this. Greg Bruhns approved the following text. Please change the text to this:

"Please visit {copyright_link}Copyright{link_end} for information pertaining to intellectual property rights and copyright infringement on Stanford Online."

**Comment by a-gene-i** [comment]
Thanks @mondiaz! I updated the text to reflect the change. @stvstnfrd @caesar2164

**Comment by caesar2164** [comment]
Looks pretty good.

@glewis17 now you just need to squash, and we can merge it.

**Review by caesar2164 — APPROVED** [review]

**Review by stvstnfrd — APPROVED** [review]

**Pull request merged**